cmd/podman/images/save.go

```diff
@@ -91,11 +97,46 @@ func save(cmd *cobra.Command, args []string) error {
 		return errors.Errorf("--compress can only be set when --format is either 'oci-dir' or 'docker-dir'")
 	}
 	if len(saveOpts.Output) == 0 {
-		fi := os.Stdout
-		if terminal.IsTerminal(int(fi.Fd())) {
+		pipeFile := "saveio"
+		pipeDir, err := ioutil.TempDir(os.TempDir(), "pipeDir")
+		if err != nil {
+			return err
+		}
+		saveOpts.Output = filepath.Join(pipeDir, pipeFile)
+		err = syscall.Mkfifo(saveOpts.Output, 0600)
+		if err != nil {
+			return errors.Wrapf(err, "error creating named pipe")
+		}
+		defer os.Remove(saveOpts.Output)
+		errc := make(chan error)
+		fpipe, err := os.OpenFile(saveOpts.Output, os.O_RDWR, os.ModeNamedPipe)
+		defer fpipe.Close()
+		if err != nil {
+			return err
+		}
+		if terminal.IsTerminal(int(fpipe.Fd())) {
```

Comment:
This should be testing the original standard output (we _know_ a FIFO is not a TTY).